### DISCUSSION TITLE
Human Friendly DNSCrypt DoH proxy resolver list in Notion

### DISCUSSION THREAD

**Discussion by nadeem49** [DNSCrypt/dnscrypt-proxy#2205] [Ideas]
I re-created a Database in Notion for the purpose of easy searching of Servers which was not possible in the official page. To make things easier to find I have added 9 Filter views. Who ever wants to maintain and contribute can volunteer.
Check out here : https://github.com/nadeem49/DNSCrypt-DOH-Public-Servers-in-Notion

**Comment by jedisct1** [comment]
This is cool!

On the [web servers list](https://dnscrypt.info/public-servers), one can click the header of a column in order to sort by that column.

But that table could indeed greatly be improved.

**Reply by nadeem49** [reply]
yeah, you can add the repo to the readme if you want. That way many will be able to access it,
